<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Rohan" name="Command line test" tests="3" failures="3" timestamp="16 Feb 2016 08:47:39 GMT" time="4.1" errors="0">
  <testcase name="listApplication" time="0.227" classname="gapp.model.dao.ApplicationDaoTest">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
	at gapp.model.dao.ApplicationDaoTest.listApplication(ApplicationDaoTest.java:24)
	at org.springframework.test.context.testng.AbstractTestNGSpringContextTests.run(AbstractTestNGSpringContextTests.java:171)
... Removed 36 stack frames]]>
    </failure>
  </testcase> <!-- listApplication -->
  <testcase name="listSubmited" time="0.014" classname="gapp.model.dao.ApplicationDaoTest">
    <failure type="org.springframework.dao.InvalidDataAccessApiUsageException" message="org.hibernate.QueryException: could not resolve property: username of: gapp.model.Users [from gapp.model.Application where student.user.username = &amp;apos;student1&amp;apos; and student.user.email = &amp;apos;student1@localhost.localdomain&amp;apos;]; nested exception is java.lang.IllegalArgumentException: org.hibernate.QueryException: could not resolve property: username of: gapp.model.Users [from gapp.model.Application where student.user.username = &amp;apos;student1&amp;apos; and student.user.email = &amp;apos;student1@localhost.localdomain&amp;apos;]">
      <![CDATA[org.springframework.dao.InvalidDataAccessApiUsageException: org.hibernate.QueryException: could not resolve property: username of: gapp.model.Users [from gapp.model.Application where student.user.username = 'student1' and student.user.email = 'student1@localhost.localdomain']; nested exception is java.lang.IllegalArgumentException: org.hibernate.QueryException: could not resolve property: username of: gapp.model.Users [from gapp.model.Application where student.user.username = 'student1' and student.user.email = 'student1@localhost.localdomain']
	at org.springframework.orm.jpa.EntityManagerFactoryUtils.convertJpaAccessExceptionIfPossible(EntityManagerFactoryUtils.java:381)
	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.translateExceptionIfPossible(AbstractEntityManagerFactoryBean.java:418)
	at org.springframework.dao.support.ChainedPersistenceExceptionTranslator.translateExceptionIfPossible(ChainedPersistenceExceptionTranslator.java:59)
	at org.springframework.dao.support.DataAccessUtils.translateIfNecessary(DataAccessUtils.java:213)
	at org.springframework.dao.support.PersistenceExceptionTranslationInterceptor.invoke(PersistenceExceptionTranslationInterceptor.java:147)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.framework.JdkDynamicAopProxy.invoke(JdkDynamicAopProxy.java:208)
	at com.sun.proxy.$Proxy34.listSubmitted(Unknown Source)
	at gapp.model.dao.ApplicationDaoTest.listSubmited(ApplicationDaoTest.java:30)
	at org.springframework.test.context.testng.AbstractTestNGSpringContextTests.run(AbstractTestNGSpringContextTests.java:171)
Caused by: java.lang.IllegalArgumentException: org.hibernate.QueryException: could not resolve property: username of: gapp.model.Users [from gapp.model.Application where student.user.username = 'student1' and student.user.email = 'student1@localhost.localdomain']
	at org.hibernate.ejb.AbstractEntityManagerImpl.convert(AbstractEntityManagerImpl.java:1374)
	at org.hibernate.ejb.AbstractEntityManagerImpl.convert(AbstractEntityManagerImpl.java:1310)
	at org.hibernate.ejb.AbstractEntityManagerImpl.createQuery(AbstractEntityManagerImpl.java:309)
	at org.springframework.orm.jpa.SharedEntityManagerCreator$SharedEntityManagerInvocationHandler.invoke(SharedEntityManagerCreator.java:291)
	at com.sun.proxy.$Proxy31.createQuery(Unknown Source)
	at gapp.model.dao.jpa.ApplicationDaoImpl.listSubmitted(ApplicationDaoImpl.java:36)
	at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:302)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.invokeJoinpoint(ReflectiveMethodInvocation.java:190)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:157)
	at org.springframework.dao.support.PersistenceExceptionTranslationInterceptor.invoke(PersistenceExceptionTranslationInterceptor.java:136)
	... 41 more
Caused by: org.hibernate.QueryException: could not resolve property: username of: gapp.model.Users [from gapp.model.Application where student.user.username = 'student1' and student.user.email = 'student1@localhost.localdomain']
	at org.hibernate.persister.entity.AbstractPropertyMapping.propertyException(AbstractPropertyMapping.java:83)
	at org.hibernate.persister.entity.AbstractPropertyMapping.toType(AbstractPropertyMapping.java:77)
	at org.hibernate.persister.entity.AbstractEntityPersister.toType(AbstractEntityPersister.java:1967)
	at org.hibernate.hql.internal.ast.tree.FromElementType.getPropertyType(FromElementType.java:313)
	at org.hibernate.hql.internal.ast.tree.FromElement.getPropertyType(FromElement.java:490)
	at org.hibernate.hql.internal.ast.tree.DotNode.getDataType(DotNode.java:632)
	at org.hibernate.hql.internal.ast.tree.DotNode.prepareLhs(DotNode.java:270)
	at org.hibernate.hql.internal.ast.tree.DotNode.resolve(DotNode.java:214)
	at org.hibernate.hql.internal.ast.tree.FromReferenceNode.resolve(FromReferenceNode.java:119)
	at org.hibernate.hql.internal.ast.tree.FromReferenceNode.resolve(FromReferenceNode.java:115)
	at org.hibernate.hql.internal.ast.HqlSqlWalker.resolve(HqlSqlWalker.java:883)
	at org.hibernate.hql.internal.antlr.HqlSqlBaseWalker.expr(HqlSqlBaseWalker.java:1267)
	at org.hibernate.hql.internal.antlr.HqlSqlBaseWalker.exprOrSubquery(HqlSqlBaseWalker.java:4552)
	at org.hibernate.hql.internal.antlr.HqlSqlBaseWalker.comparisonExpr(HqlSqlBaseWalker.java:4022)
	at org.hibernate.hql.internal.antlr.HqlSqlBaseWalker.logicalExpr(HqlSqlBaseWalker.java:2102)
	at org.hibernate.hql.internal.antlr.HqlSqlBaseWalker.logicalExpr(HqlSqlBaseWalker.java:2027)
	at org.hibernate.hql.internal.antlr.HqlSqlBaseWalker.whereClause(HqlSqlBaseWalker.java:796)
	at org.hibernate.hql.internal.antlr.HqlSqlBaseWalker.query(HqlSqlBaseWalker.java:597)
	at org.hibernate.hql.internal.antlr.HqlSqlBaseWalker.selectStatement(HqlSqlBaseWalker.java:301)
	at org.hibernate.hql.internal.antlr.HqlSqlBaseWalker.statement(HqlSqlBaseWalker.java:249)
	at org.hibernate.hql.internal.ast.QueryTranslatorImpl.analyze(QueryTranslatorImpl.java:250)
	at org.hibernate.hql.internal.ast.QueryTranslatorImpl.doCompile(QueryTranslatorImpl.java:185)
	at org.hibernate.hql.internal.ast.QueryTranslatorImpl.compile(QueryTranslatorImpl.java:138)
	at org.hibernate.engine.query.spi.HQLQueryPlan.<init>(HQLQueryPlan.java:104)
	at org.hibernate.engine.query.spi.HQLQueryPlan.<init>(HQLQueryPlan.java:79)
	at org.hibernate.engine.query.spi.QueryPlanCache.getHQLQueryPlan(QueryPlanCache.java:168)
	at org.hibernate.internal.AbstractSessionImpl.getHQLQueryPlan(AbstractSessionImpl.java:222)
	at org.hibernate.internal.AbstractSessionImpl.createQuery(AbstractSessionImpl.java:200)
	at org.hibernate.internal.SessionImpl.createQuery(SessionImpl.java:1739)
	at org.hibernate.ejb.AbstractEntityManagerImpl.createQuery(AbstractEntityManagerImpl.java:301)
	... 56 more
... Removed 44 stack frames]]>
    </failure>
  </testcase> <!-- listSubmited -->
  <testcase name="getDepartments" time="0.03" classname="gapp.model.dao.DepartmentDaoTest">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
	at gapp.model.dao.DepartmentDaoTest.getDepartments(DepartmentDaoTest.java:24)
	at org.springframework.test.context.testng.AbstractTestNGSpringContextTests.run(AbstractTestNGSpringContextTests.java:171)
... Removed 36 stack frames]]>
    </failure>
  </testcase> <!-- getDepartments -->
</testsuite> <!-- Command line test -->
